### DISCUSSION TITLE
可以在快捷方式中添加参数来达到启动时缩小到托盘的效果吗?

### DISCUSSION THREAD

**Discussion by MoTingFeng** [shion-app/shion#21] [Ideas]
因为是使用定时方式启动的软件,希望可以通过控制快捷方式的参数达到启动时缩小到托盘的效果

**Comment by hanaTsuk1** [comment]
不太明白`快捷方式添加参数`想说明什么
我可以理解为就是设置`开机启动默认不显示窗口`吗

**Reply by MoTingFeng** [reply]
是`启动软件`时默认不显示窗口,据我所知有些软件可以在快捷方式的的目标路径后添加参数实现这一点

**Reply by hanaTsuk1** [reply]
那我加个设置选项就行了  不用那么麻烦

**Reply by MoTingFeng** [reply]
谢谢
